consensus/utils_test.go

```diff
@@ -390,12 +391,16 @@ func createDummyBLSPrivateKey() *bls.PrivateKey {
 }
 
 func createDummyBlockHash() *blockHash {
+	byteArray := [32]byte{}
+	copy(byteArray[:], generateRandomBytes(32))
+
 	return &blockHash{
-		value: [32]byte{
-			0x0, 0x1, 0x2, 0x3, 0x4, 0x5, 0x6, 0xf,
-			0x8, 0x9, 0xa, 0xb, 0xc, 0xd, 0xe, 0xf,
-			0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16,
-			0x17, 0x18, 0x19, 0x1a, 0x1b, 0x1c, 0x1d,
-		},
+		value: byteArray,
 	}
 }
+
+func generateRandomBytes(numBytes int) []byte {
+	randomBytes := make([]byte, numBytes)
+	rand.Read(randomBytes)
+	return randomBytes
+}
```

Comment:
I personally don't like using unseeded randomness because you can't repro. What I found more efficient is picking a random number and initializing rand like this:
```
	seed := int64(993)
	rand := rand.New(rand.NewSource(seed))
```
This way if you have an error it'll be easier to debug. Also it's convenient to define randomness in the parent test and pass it down when needed. So if you really want to stress-test your code you could iterate through seeds on the parent test.

Reply:
Not a priority here given how simple the usage of createBlockHash will be for tests. What I need here is a dead simple self-contained function that returns pseudo-random block hashs without repeating values. Reproducibility of the generated value across test executions isn't important.